pkg/controller/controller_utils.go

```diff
@@ -602,7 +602,7 @@ func (r RealPodControl) DeletePod(namespace string, podID string, object runtime
 	klog.V(2).InfoS("Deleting pod", "controller", accessor.GetName(), "pod", klog.KRef(namespace, podID))
 	if err := r.KubeClient.CoreV1().Pods(namespace).Delete(context.TODO(), podID, metav1.DeleteOptions{}); err != nil {
 		if apierrors.IsNotFound(err) {
-			klog.V(4).Infof("pod %v/%v has already been deleted.", namespace, podID)
+			klog.V(4).InfoS("Pod has already been deleted.", "podNamespace", namespace, "pod", podID)
```

Comment:
klog.KRef or klog.KObj？

Reply:
klog.kObj wouldn't work and I need the pod name for klog.KRef, which podID sounds different from?

Reply:
Is it okay to keep the format of L602?
Is it works?